integration-tests/actions/actions.go

```diff
@@ -369,7 +369,7 @@ func ReturnFunds(chainlinkNodes []*client.ChainlinkK8sClient, blockchainClient b
 			}
 			err = blockchainClient.ReturnFunds(decryptedKey.PrivateKey)
 			if err != nil {
-				return err
+				log.Error().Err(err).Str("Address", fundedKeys[0].Address).Msg("Error returning funds from Chainlink node")
```

Comment:
Does Log.Error() stop the test  or just logs error?

Reply:
Just logs an error. This is at the return funds step, so if we can't on one node, we should keep moving on and try the next one instead of stopping.